src/exec.js

```diff
@@ -98,8 +113,9 @@ function execSync(cmd, opts) {
     // sleepFile is used as a dummy I/O op to mitigate unnecessary CPU usage
     // (tried many I/O sync ops, writeFileSync() seems to be only one that is effective in reducing
     // CPU usage, though apparently not so much on Windows)
-    while (!fs.existsSync(codeFile)) { updateStdout(); fs.writeFileSync(sleepFile, 'a'); }
-    while (!fs.existsSync(stdoutFile)) { updateStdout(); fs.writeFileSync(sleepFile, 'a'); }
+    while (!fs.existsSync(codeFile)) { updateStream(stdoutFile); fs.writeFileSync(sleepFile, 'a'); }
```

Comment:
Which does it duplicate? That's meant to replace the original `!fs.existsSync(codeFile)` line. It differs from the next line, which uses `!fs.existsSync(stdoutFile)`